pom.xml

```diff
@@ -28,7 +39,17 @@
             <version>6.6.0</version>
```

Comment:
please check the duplicate of dependence

Reply:
there are no duplicates
one dependency is cucumber-java, and the other is cucumber-junit

Reply:
sorry, my mistake